2.data.sql

```diff
@@ -0,0 +1,21 @@
+use CostumeDB
+go 
+delete Sales
+go
+
+insert Sales(CustomerName, CostumesBought, CostumeSize, Quantity, PriceSoldPerCostume, DateBought, DateSold)
+select 'Chana Goldberg', 'Artist', 'XS', 2, 20, 'Feb 14 2020', 'Apr 2 2020'
```

Comment:
AS: -2 Dates should be in correct date format: yyyy-mm-dd or other date format.

Reply:
changed